products/silverstripe.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Silverstripe CMS
+category: server-app
+permalink: /silverstripe
+alternate_urls:
+- /silverstripe-cms
+- /silverstripe-framework
+versionCommand: composer info silverstripe/cms
+releaseImage: https://www.silverstripe.org/assets/Uploads/_resampled/ResizedImageWzYwMCwzOTdd/CMS-5-Support-Timeline-May-2023.png 
+releasePolicyLink: https://www.silverstripe.org/software/roadmap/
+changelogTemplate: https://github.com/silverstripe/silverstripe-cms/releases/__LATEST__
+activeSupportColumn: true
+releaseDateColumn: true
+releaseColumn: true
+eolWarnThreshold: 182
+
+auto:
+-   git: https://github.com/silverstripe/silverstripe-cms.git
+
+identifiers:
+-   purl: pkg:composer/silverstripe/cms
+-   purl: pkg:github/silverstripe/cms
+
+releases:
+-   releaseCycle: "5.0"
+    support: 2028-04-01
+    eol: 2028-04-01
+    latest: "5.0.4"
+    latestReleaseDate: 2023-05-31
+    releaseDate: 2023-05-01
```

Comment:
Since the dates for 5.0 are currently undecided, and we're marking this as `5.0` (not `5`), lets just mark it as "currently supported" using `true/false`. Once the dates are announced, we can adjust accordingly.

```suggestion
-   releaseCycle: "5.0"
    support: true
    eol: false
    latest: "5.0.4"
    latestReleaseDate: 2023-05-31
    releaseDate: 2023-05-01
```